### DISCUSSION TITLE
I pushed with a non-collaborator account on a private repo, and it worked

### DISCUSSION THREAD

**Discussion by GioAgu17** [community/community#86711] [Repositories]
![image](https://github.com/community/community/assets/23059918/19bed93b-3158-4262-9afe-9274ac7f3af1)
As you can see, I pushed with the account tech7-formatori on this repo.
The problem is that the repo is private, its owner is different than tech7-formatori (as you can see from profile icons) and it doesn't have any collaborators.

How is that possible?

PS: I pushed via VSCode VCS

**Comment by kmadaminjanova0508** [comment]
If you pushed commits to a private repository, I think you are using SSH for authentication. If the SSH key on your machine is associated with the tech7-formatori account, you might have this kind of situation, or just check if you have stored credentials on your local machine. It's possible that the credentials stored on your system are associated with the tech7-formatori account. The stored credentials might also be used automatically when you perform actions like pushing from VSCode.. or If you are using HTTPS and an access token, check if the access token used by VSCode is associated with the tech7-formatori account that can also be one reason in your case, anyway it is somehow strange if you've verified that none of the above reasons are causing the issue, I think it would be better  reaching out to GitHub Support for further assistance. They will provide more specific guidance based on the details of your situation. :)
